examples/layers/macos/.gitignore

```diff
@@ -0,0 +1,7 @@
+# Flutter-related
```

Comment:
I see some slipped through the cracks but we generally don't check in these example app .gitignore files.
Update https://github.com/flutter/flutter/blob/master/.gitignore if there are more patterns to exclude.